cellprofiler/modules/relateobjects.py

```diff
@@ -108,7 +113,7 @@ def create_settings(self):
             doc="""\
 Choose the method to calculate distances of each child to its parent.
```

Comment:
Would you be willing to add a one sentence use case? For example:

"For example, these measurements can tell you whether nuclear speckles are located more closely to the center of the nucleus or to the nuclear periphery".